koa/koa.d.ts

```diff
@@ -16,99 +20,101 @@ declare module "koa" {
   import * as http from "http";
   import * as net from "net";
 
-  interface IContext extends IRequest, IResponse {
-      body?: any;
-      request?: IRequest;
-      response?: IResponse;
-      originalUrl?: string;
-      state?: any;
-      name?: string;
-      cookies?: any;
-      writable?: Boolean;
-      respond?: Boolean;
-      app?: Koa;
-      req?: http.IncomingMessage;
-      res?: http.ServerResponse;
-      onerror(err: any): void;
-      toJSON(): any;
-      inspect(): any;
-      throw(): void;
-      assert(): void;
-  }
+  module Koa {
+    export interface IContext extends IRequest, IResponse {
```

Comment:
please remove `I` prefix from interface.
from http://www.typescriptlang.org/Handbook#writing-dts-files 

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.